wordle-web/src/scripts/letter.ts

```diff
@@ -0,0 +1,32 @@
+// Class that represents a letter in the wordle word and status
+// of the letter in the wordle word
+export enum LetterStatus {
+    NotGuessed = 0,
+    Correct,
+    Misplaced,
+    Wrong
+}
+
+
+export class Letter {
+    letter?: string;
+    status: LetterStatus = LetterStatus.NotGuessed;
+
+    constructor (letter: string) {
+        this.letter = letter;
+    }
+
+    get color() {
+        switch (this.status) {
+            case LetterStatus.Correct:
+                return 'green';
+            case LetterStatus.Misplaced:
+                return 'orange';
+            case LetterStatus.Wrong:
+                return 'red';
+            case LetterStatus.NotGuessed:
+                return 'grey';
```

Comment:
I think in class we made this a default. You can check how Meg did it in the main branch. Not a huge deal in this case I feel like.

Reply:
That is a cleaner way to do it, I like that better.